main.cpp

```diff
@@ -48,7 +48,7 @@ void Backup()
 
 int main()
 {
-	SetConsoleTitle("totally not pasted injector :)");
+	SetConsoleTitle(L"totally not pasted injector :)");

```

Comment:
Hi i have no knowledge with C++ but what does the L next to the ( mean?

Reply:
it means that the type of string is wchar_t or something like this